test/integration/evictions/evictions_test.go

```diff
@@ -157,73 +164,76 @@ func TestConcurrentEvictionRequests(t *testing.T) {
 		}
 	}
 
-	waitToObservePods(t, podInformer, 10)
+	waitToObservePods(t, podInformer, numOfEvictions)
 
 	pdb := newPDB()
 	if _, err := clientSet.Policy().PodDisruptionBudgets(ns.Name).Create(pdb); err != nil {
 		t.Errorf("Failed to create PodDisruptionBudget: %v", err)
 	}
 
-	waitPDBStable(t, clientSet, 10, ns.Name, pdb.Name)
-
-	doneCh := make(chan bool, 10)
-	errCh := make(chan error, 1)
-	// spawn 10 goroutine to concurrently evict the pods
-	for i := 0; i < 10; i++ {
-		go func(id int, doneCh chan bool, errCh chan error) {
-			evictionName := fmt.Sprintf(podNameFormat, id)
-			eviction := newEviction(ns.Name, evictionName, deleteOption)
-
+	waitPDBStable(t, clientSet, numOfEvictions, ns.Name, pdb.Name)
+
+	doneCount := int32(0)
```

Comment:
we may want to define this variable closer to where it's used and possibly use `var doneCount int32 = 0` to avoid the explicit cast. Also a `uint32` is probably fine